### DISCUSSION TITLE
On the links with icons block should we review and refactor this block

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#19] [Enhancement Requests]
We had a lot of small issues with this block and an overhaul is probably due.
https://github.com/amnestywebsite/amnesty-wp-theme/issues/2373

**Comment by jaymcp** [comment]
100%. We could action this as part of https://github.com/amnestywebsite/humanity-theme/issues/6, potentially.

**Comment by AmnestyAM** [comment]
Also discussed here https://github.com/amnestywebsite/amnesty-wp-theme/discussions/2387
